src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,108 @@
+import classNames from 'classnames';
+import React, {
+  useContext, useEffect, useRef, useState,
+} from 'react';
+import { Todo } from '../../types/Todo';
+import { TodosContext } from '../../contextes/TodosContext';
+
+type Props = {
+  todo: Todo
+};
+
+export const TodoItem: React.FC<Props> = ({ todo }) => {
+  const [editing, setEditing] = useState(false);
```

Comment:
Give a name to Boolean vars with is, has etc.
```suggestion
  const [isEditing, setIsEditing] = useState(false);
```